Automattic/woocommerce-payments
# Add a "No transactions to display" message when transaction table is empty

~Fixes~ Does not address #97 (yet).

#### Changes proposed in this Pull Request

* When there are no transactions available, display an empty table informing the user that there are `No transactions to display`:

<img width="1455" alt="image" src="https://user-images.githubusercontent.com/13835680/60445232-54576680-9c0e-11e9-937c-bb52da9e5f53.png">

* While loading the transactions list, display the "loading" version of the table (not really a change, more a requirement; the table worked like this before):

<img width="1455" alt="image" src="https://user-images.githubusercontent.com/13835680/60445360-97b1d500-9c0e-11e9-9976-bfa32e3339a8.png">

#### Testing instructions



Make sure the following requirements are met:

* Display data when data is available.
* Display `No transactions to display` message when:
    * The `transactions` variable is set to `[]`.
    * The `rows` variable is set to either `[]` or `undefined`.

-------------------
